app/routes/circulars/circulars.lib.ts

```diff
@@ -116,6 +126,7 @@ export const validSubjectKeywords = [
   'Baksan Neutrino Observatory Alert',
   'CALET',
   'Chandra',
+  'Duplicate Submission',
```

Comment:
Move these two keyword additions to a separate PR.

Reply:
I don't understand. Do people request changes to a Circular by submitting a Circular with these magic works in the subject?

Reply:
Ah I see, no I think I am seeing an error in my implementation. I don't want the users to submit circulars where the value is literally 'Duplicate Submission', I was checking that the string was valid, but it shouldn't be the case that these are just considered valid keywords. 

In the [big PR](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1425) I had a couple simplified workflows that would be shortcuts for mods to just click a button and it would handle the workflow for duplicates/mistaken submissions

I have some ideas for how to improve this, the functionality as I added it is wrong, I'll update this shortly